packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -55,36 +55,50 @@ const defaultMuiTheme: ThemeOptions = {
     MuiDialog: {
       defaultProps: {
         PaperProps: {
-          sx: {
+          sx: theme => ({
             borderRadius: '0px',
-            padding: '35px',
+            padding: theme.spacing(5.5),
             alignSelf: 'flex-start',
-          },
+          }),
         },
       },
     },
     MuiDialogTitle: {
       styleOverrides: {
         root: ({ theme }) => ({
-          borderBottom: `1px solid ${theme.palette.grey[300]}`,
-          padding: '0px 0px 20px',
-          marginBottom: '20px',
+          fontSize: '24px',
+          fontWeight: 700,
+          paddingLeft: '0px',
+          paddingRight: '0px',
+          paddingBottom: theme.spacing(3),
         }),
       },
     },
     MuiDialogContent: {
+      defaultProps: {
+        dividers: true,
+      },
```

Comment:
Didn't know this prop existed. It lets us remove the border styles on MuiDialogTitle and MuiDialogContent